lua/lint/parser.lua

```diff
@@ -72,7 +72,7 @@ function M.from_pattern(pattern, groups, severity_map, defaults)
   defaults = defaults or {}
   return function(output, bufnr)
     local diagnostics = {}
-    local buffer_path = vim.api.nvim_buf_get_name(bufnr)
+    local buffer_path = vim.fn.fnamemodify(vim.api.nvim_buf_get_name(bufnr), ":~:.")
```

Comment:
Sure that all linters do this? I think it would be safer to also apply the `fnamemodify` call further down in the `entries['file'] == buffer_path` - that way it should also work if the linter reports the absolute path.

Reply:
Ah, right. That makes sense.